include/eve/module/core/regular/impl/is_less.hpp

```diff
@@ -0,0 +1,42 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+namespace eve::detail
+{
+  template<typename T, typename U, callable_options O>
+  EVE_FORCEINLINE constexpr common_logical_t<T, U> is_less_(EVE_REQUIRES(cpu_), O const& o, T a, U b) noexcept
```

Comment:
I'm concerned about implicit coinvercions. 

I think that our implicit conversions might prove problematic here.

Let's restrict this to the same element_type at least.

Reply:
we need to support `is_less(wide<float>, int)`

Reply:
The implicit conversions should be done by elementwise callable, not here. 
I am concerned that this might get called instead of being prior converted.